core/tauri-macros/src/command/wrapper.rs

```diff
@@ -234,14 +261,22 @@ fn body_async(
   invoke: &Invoke,
   attributes: &WrapperAttributes,
 ) -> syn::Result<TokenStream2> {
-  let Invoke { message, resolver } = invoke;
-  parse_args(function, message, attributes).map(|args| {
+  let Invoke {
+    id,
+    message,
+    resolver,
+  } = invoke;
+  parse_args(id, function, message, attributes).map(|args| {
     quote! {
+      use tracing::Instrument;
+
+      let span = tracing::debug_span!("ipc::request::run", id = #id.0);
```

Comment:
```suggestion
      let span = tracing::debug_span!("ipc::request::handler", id = #id.0);
```

See previous comment